### PR TITLE
Allow users to set an active deck

### PR DESCRIPTION
**Pull request by Vinnehboom**

### CODE REVIEW

File: components/DeckItem.tsx [hunk 1]
```diff
@@ -28,20 +29,7 @@ export const DeckItem = ({ deck }: DeckListPropTypes) => {
...
-                source={{
-                  uri: `https://limitlesstcg.s3.us-east-2.amazonaws.com/pokemon/gen9/${icon}.png`,
-                }}
-                minHeight="100%"
-                resizeMode="stretch"
-                alt={icon}
-                size="2xs"
-              />
-            ))}
+          <ArchetypeIcons deck={deck} />
```
Comment (dennishavermans) [inline components/DeckItem.tsx:32]: Very clean!

File: screens/LandingScreen.tsx [hunk 2]
```diff
@@ -1,43 +1,46 @@
...
+  const deck = activeDeck?.deckId;
   return (
-    <View style={styles.container}>
-      <Text>
-        {t("TYPES.USER.EMAIL")}: {user.email}
-      </Text>
-      <TouchableOpacity onPress={signOut} style={styles.button}>
-        <Text style={styles.buttonText}> {t("LANDING_SCREEN.SIGN_OUT")} </Text>
-      </TouchableOpacity>
+    <View style={LandingScreenStyle.container}>
```
Comment (dennishavermans) [inline screens/LandingScreen.tsx:20]: Do we need the wrapper **<View>** here? Looks like you could just move the **LandingScreenStyle.container** style down to the **<ScrollView>**

File: screens/LandingScreen.tsx [hunk 3]
```diff
@@ -1,43 +1,46 @@
...
+            <VStack width={"100%"} justifyContent={"center"} display={"flex"}>
+              <Text style={LandingScreenStyle.activeDeckTitle}>{t("LANDING_SCREEN.ACTIVE_DECK.TITLE")}</Text>
+              <HStack style={LandingScreenStyle.activeDeck}>
+                <Text style={LandingScreenStyle.activeDeckName}>{deck.name}</Text>
+                <ArchetypeIcons deck={deck} size={"xs"} />
+              </HStack>
+            </VStack>
+          </Box>
+        ) : (
+          <></>
```
Comment (dennishavermans) [inline screens/LandingScreen.tsx:37]: Is this intended? If not you should consider re writing the check.

Comment (Vinnehboom) [reply screens/LandingScreen.tsx:37]: It is, but you raise a good point, maybe we want to guide the user to their decks to set an active deck here if none is set yet!

**Review by dennishavermans — APPROVED** [review]

**Pull request merged**